src/mpid/ch4/generic/am/mpidig_am_send_utils.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+
+#ifndef MPIDIG_AM_SEND_UTILS_H_INCLUDED
+#define MPIDIG_AM_SEND_UTILS_H_INCLUDED
+
+/* This file is for supporting routines used for pipelined data send. These routines mainly is for
+ * managing the send request counters, completion counters and DT refcount */
+
+/* prepare the send counters for size of data and datatype that we need to refcount */
+MPL_STATIC_INLINE_PREFIX void MPIDIG_send_init(MPIR_Request * sreq, MPI_Datatype datatype,
```

Comment:
We should change the name to avoid confusion with `MPI_Send_init`.

Reply:
OK. I have renamed them as `MPIDIG_am_send_*`. We should also rename `MPIDIG_recv_init` to avoid confusion with `MPI_Recv_init`, probably with a separate PR.

Also fixed the error suppression.